counterfab/src/main/res/values/attrs.xml

```diff
@@ -2,5 +2,11 @@
 <resources>
     <declare-styleable name="CounterFab">
         <attr name="badgeBackgroundColor" format="color" />
+        <attr name="badgePosition" format="enum">
+            <enum name="RightAndTop" value="0" />
+            <enum name="LeftAndBottom" value="1" />
+            <enum name="LeftAndTop" value="2" />
+            <enum name="RightAndBottom" value="3" />
```

Comment:
Could you remove the `And` word of the enums?
Having just `RightBottom` for example.

Reply:
ok